### PR TITLE
Fix/4601 better messages when author and user are the same in notifications

### PR DESCRIPTION
**Pull request by raphj**
#4601

NOTE: I don't think tests are worth it here.

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [X] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [X] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by gbonaspetti — APPROVED** [review]

File: frontend/i18next.scanner/de/translation.json [hunk 1]
```diff
@@ -490,5 +487,12 @@
...
   "{{author}} made {{numberOfContribution}} contributions": "{{author}} hat {{numberOfContribution}} Beiträge geleistet",
-  "in {{numberOfWorkspaces}} spaces": "in {{numberOfWorkspaces}} Bereichen"
+  "in {{numberOfWorkspaces}} spaces": "in {{numberOfWorkspaces}} Bereichen",
+  "{{author}} created their account": "{{author}} hat sein Konto erstellt",
+  "{{author}} modified their account": "{{author}} hat sein Konto geändert",
+  "{{author}} deleted their account": "{{author}} hat sein Konto gelöscht",
+  "{{author}} restored their account": "{{author}} hat sein Konto wiederhergestellt",
+  "{{author}} added themself to a space": "{{author}} hat sich selbst zu einem Raum hinzugefügt",
+  "{{author}} modified their role in a space": "{{author}} hat seine Rolle in einem Bereich geändert",
+  "{{author}} removed themself from a space": "{{author}} hat sich selbst aus einem Bereich entfernt"
```
Comment (raphj) [inline frontend/i18next.scanner/de/translation.json:497]:
```suggestion
  "{{author}} modified their role in a space": "{{author}} hat seine Rolle in einem Bereich geändert"
```

File: frontend/i18next.scanner/en/translation.json [hunk 2]
```diff
@@ -490,5 +487,12 @@
...
   "Set your username": "Set your username",
-  "in {{numberOfWorkspaces}} spaces": "in {{numberOfWorkspaces}} spaces"
+  "in {{numberOfWorkspaces}} spaces": "in {{numberOfWorkspaces}} spaces",
+  "{{author}} created their account": "{{author}} created their account",
+  "{{author}} modified their account": "{{author}} modified their account",
+  "{{author}} deleted their account": "{{author}} deleted their account",
+  "{{author}} restored their account": "{{author}} restored their account",
+  "{{author}} added themself to a space": "{{author}} added themself to a space",
+  "{{author}} modified their role in a space": "{{author}} modified their role in a space",
+  "{{author}} removed themself from a space": "{{author}} removed themself from a space"
```
Comment (raphj) [inline frontend/i18next.scanner/en/translation.json:497]:
```suggestion
  "{{author}} modified their role in a space": "{{author}} modified their role in a space"
```

File: frontend/i18next.scanner/fr/translation.json [hunk 3]
```diff
@@ -490,5 +487,12 @@
...
   "Set your username": "Définissez votre identifiant",
-  "in {{numberOfWorkspaces}} spaces": "dans {{numberOfWorkspaces}} espaces"
+  "in {{numberOfWorkspaces}} spaces": "dans {{numberOfWorkspaces}} espaces",
+  "{{author}} created their account": "{{author}} a créé son compte",
+  "{{author}} modified their account": "{{author}} a modifié son compte",
+  "{{author}} deleted their account": "{{author}} a supprimé son compte",
+  "{{author}} restored their account": "{{author}} a restauré son compte",
+  "{{author}} added themself to a space": "{{author}} s'est ajouté dans l'espace",
+  "{{author}} modified their role in a space": "{{author}} a modifié son rôle dans un espace",
+  "{{author}} removed themself from a space": "{{author}} a quitté l'espace"
```
Comment (raphj) [inline frontend/i18next.scanner/fr/translation.json:497]:
```suggestion
  "{{author}} modified their role in a space": "{{author}} a modifié son rôle dans un espace"
```

File: frontend/i18next.scanner/pt/translation.json [hunk 4]
```diff
@@ -490,5 +487,12 @@
...
   "{{author}} requested access to a space": "{{author}} solicitou acesso a um espaço",
-  "Tags": "Etiquetas"
+  "Tags": "Etiquetas",
+  "{{author}} created their account": "{{author}} criou a sua conta",
+  "{{author}} modified their account": "{{author}} modificou a sua conta",
+  "{{author}} deleted their account": "{{author}} apagou a sua conta",
+  "{{author}} restored their account": "{{author}} restaurou a sua conta",
+  "{{author}} added themself to a space": "{{author}} adicionou-se a um espaço",
+  "{{author}} modified their role in a space": "{{author}} modificou a sua função num espaço",
+  "{{author}} removed themself from a space": "{{author}} retirou-se de um espaço"
```
Comment (raphj) [inline frontend/i18next.scanner/pt/translation.json:497]:
```suggestion
  "{{author}} modified their role in a space": "{{author}} modificou a sua função num espaço"
```

**Review by MariePrandstetter — APPROVED** [review]

**Comment by raphj** [comment]
To test, try both with web.notifications.excluded = workspace.modified; and with the default value.

**Comment by MariePrandstetter** [comment]
It works fine, but on grouped notifications it still shows "X modified X's account". But I think this is ok

**Comment by MariePrandstetter** [comment]
tested on branch

**Pull request merged**